docs/src/content/docs/fr/reference/overrides.md

```diff
@@ -0,0 +1,374 @@
+//      ^ type: boolean
+---
+```
+
+Cela vous permettra d'obtenir de l'autocomplétion et un typage lors de l'utilisation de `Astro.props`.
+
+### Props
+
+Starlight passera les props suivantes à vos composants personnalisés.
+
+#### `dir`
+
+**Type:** `'ltr' | 'rtl'`
+
+Le sens d'écriture de la page.
+
+#### `lang`
+
+**Type:** `string`
+
+L’étiquette d’identification BCP-47 pour la langue de la page, par exemple `en`, `zh` ou `pt-BR`.
+
+#### `locale`
+
+**Type:** `string | undefined`
+
+Le chemin de base utilisé pour servir une langue. `undefined` pour les slugs de la locale racine.
+
+#### `slug`
+
+**Type:** `string`
+
+Le slug de la page généré à partir du nom du fichier du contenu.
+
+#### `id`
+
+**Type:** `string`
+
+L'identifiant unique de cette page basé sur le nom du fichier du contenu.
+
+#### `isFallback`
+
+**Type:** `true | undefined`
+
+`true` si cette page n'est pas traduite dans la langue actuelle et utilise le contenu de la langue par défaut en tant que repli.
+Utilisé uniquement dans les sites multilingues.
+
+#### `entryMeta`
+
+**Type:** `{ dir: 'ltr' | 'rtl'; lang: string }`
+
+Métadonnées de la locale pour le contenu de la page. Peut être différent des valeurs de locale de premier niveau lorsque la page utilise un contenu de repli.
+
+#### `entry`
+
+L'entrée de la collection de contenu Astro pour la page courante.
+Inclut les valeurs du frontmatter pour la page courante dans `entry.data`.
+
+```ts
+entry: {
+  data: {
+    title: string;
+    description: string | undefined;
+    // etc.
+  }
+}
+```
+
+Pour en savoir plus sur le format de cet objet, consultez la [référence du type d'entrée de collection](https://docs.astro.build/fr/reference/api-reference/#collection-entry-type).
+
+#### `sidebar`
+
+**Type:** `SidebarEntry[]`
+
+Les entrées de la barre latérale de navigation du site pour cette page.
+
+#### `hasSidebar`
+
+**Type:** `boolean`
+
+Indique si la barre latérale est affichée sur cette page.
+
+#### `pagination`
+
+**Type:** `{ prev?: Link; next?: Link }`
+
+Liens vers la page précédente et suivante dans la barre latérale si celle-ci est activée.
+
+#### `toc`
+
+**Type:** `{ minHeadingLevel: number; maxHeadingLevel: number; items: TocItem[] } | undefined`
+
+Table des matières de la page courante si celle-ci est activée.
+
+#### `headings`
+
+**Type:** `{ depth: number; slug: string; text: string }[]`
+
+Un tableau de toutes les en-têtes Markdown extraites de la page courante.
+Utilisez [`toc`](#toc) à la place si vous souhaitez construire une table des matières qui respecte les options de configuration de Starlight.
+
+#### `lastUpdated`
+
+**Type:** `Date | undefined`
+
+Un objet JavaScript de type `Date` représentant la date de dernière mise à jour de cette page si cette fonctionnalité est activée.
+
+#### `editUrl`
+
+**Type:** `URL | undefined`
+
+Un objet `URL` de l'adresse où cette page peut être modifiée si cette fonctionnalité est activée.
+
+---
+
+## Composants
+
+### Métadonnées
```

Comment:
"head" and "header" are usually translated both using "en-tête" which would be annoying in this page to use the same word so I used "Métadonnées" like on [MDN](https://developer.mozilla.org/fr/docs/Web/HTML/Element/head).